client/src/components/layouts/dashboardLayout/DashboardLayoutRoute.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import DashboardLayout from './DashboardLayout';
+
+const DashboardLayoutRoute = ({ component: Component, ...rest}) => {
+    //get 'component' and renamed to 'Component', any other 'props' renamed to 'rest'
+    return (
+        <Route 
```

Comment:
Is this part what you meant by nested route?

Reply:
yes

Reply:
Well I think its not really nested route, just routes with different layouts